CHANGELOG.md

```diff
@@ -1,6 +1,5 @@
 ## [0.1.1] - <Set Date>
```

Comment:
One last change. Since you're adding new features, update the version to 0.2. Try and use semantic versioning every where.